reactive/flannel.py

```diff
@@ -197,7 +207,7 @@ def configure_network(etcd):
         return False
 
 
-@when('config.changed.cidr')
+@when('config.changed.cidr', 'config.changed.port', 'config.changed.vni')
```

Comment:
This needs to be `@when_any`, not `@when`. As it is now, it will only trigger if *all* of the configs are changed at the same time.